src/main/webapp/js/maps-page-loader.js

```diff
@@ -0,0 +1,259 @@
+					"elementType": "geometry",
+					"stylers": [
+						{
+							"color": "#283d6a"
+						}
+					]
+				},
+				{
+					"featureType": "poi",
+					"elementType": "labels.text.fill",
+					"stylers": [
+						{
+							"color": "#6f9ba5"
+						}
+					]
+				},
+				{
+					"featureType": "poi",
+					"elementType": "labels.text.stroke",
+					"stylers": [
+						{
+							"color": "#1d2c4d"
+						}
+					]
+				},
+				{
+					"featureType": "poi.park",
+					"elementType": "geometry.fill",
+					"stylers": [
+						{
+							"color": "#023e58"
+						}
+					]
+				},
+				{
+					"featureType": "poi.park",
+					"elementType": "labels.text.fill",
+					"stylers": [
+						{
+							"color": "#3C7680"
+						}
+					]
+				},
+				{
+					"featureType": "road",
+					"elementType": "geometry",
+					"stylers": [
+						{
+							"color": "#304a7d"
+						}
+					]
+				},
+				{
+					"featureType": "road",
+					"elementType": "labels.text.fill",
+					"stylers": [
+						{
+							"color": "#98a5be"
+						}
+					]
+				},
+				{
+					"featureType": "road",
+					"elementType": "labels.text.stroke",
+					"stylers": [
+						{
+							"color": "#1d2c4d"
+						}
+					]
+				},
+				{
+					"featureType": "road.highway",
+					"elementType": "geometry",
+					"stylers": [
+						{
+							"color": "#2c6675"
+						}
+					]
+				},
+				{
+					"featureType": "road.highway",
+					"elementType": "geometry.stroke",
+					"stylers": [
+						{
+							"color": "#255763"
+						}
+					]
+				},
+				{
+					"featureType": "road.highway",
+					"elementType": "labels.text.fill",
+					"stylers": [
+						{
+							"color": "#b0d5ce"
+						}
+					]
+				},
+				{
+					"featureType": "road.highway",
+					"elementType": "labels.text.stroke",
+					"stylers": [
+						{
+							"color": "#023e58"
+						}
+					]
+				},
+				{
+					"featureType": "transit",
+					"elementType": "labels.text.fill",
+					"stylers": [
+						{
+							"color": "#98a5be"
+						}
+					]
+				},
+				{
+					"featureType": "transit",
+					"elementType": "labels.text.stroke",
+					"stylers": [
+						{
+							"color": "#1d2c4d"
+						}
+					]
+				},
+				{
+					"featureType": "transit.line",
+					"elementType": "geometry.fill",
+					"stylers": [
+						{
+							"color": "#283d6a"
+						}
+					]
+				},
+				{
+					"featureType": "transit.station",
+					"elementType": "geometry",
+					"stylers": [
+						{
+							"color": "#3a4762"
+						}
+					]
+				},
+				{
+					"featureType": "water",
+					"elementType": "geometry",
+					"stylers": [
+						{
+							"color": "#0e1626"
+						}
+					]
+				},
+				{
+					"featureType": "water",
+					"elementType": "labels.text.fill",
+					"stylers": [
+						{
+							"color": "#4e6d70"
+						}
+					]
+				}
+			],
+			{name: 'Night'});
+
+	// Create a map object, and include the MapTypeId to add
+	// to the map type control.
+	var map = new google.maps.Map(document.getElementById('map'), {
```

Comment:
The element ID 'map' here is quite hidden in the code. If we were to change the id in the HTML file, highly probable that we will forget to change it here. I'd recommend defining a file constant (a constant defined at the top of the file, named IN_ALL_CAPS, that defines this element ID and use that constant here. e.g.

`const MAP_ELEMENT_ID = 'map';`

Reply:
Silly question, but do I declare it above or in the createMap function (I'll be changing the name to initConfigMap)?

Reply:
No such thing as a silly question. You should declare it above, at the top of the file.